core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -428,11 +428,13 @@ export class RteHelperModalComponent {
               this.tmpCustomizationArgs[i - 1].value;
           }
         }
-        (
-          customizationArgsDict as {
-            [Prop in CustomizationArgsNameAndValueArray[number]['name']]: CustomizationArgsNameAndValueArray[number]['value'];
-          }
-        )[caName] = this.tmpCustomizationArgs[i].value;
+        if (this.tmpCustomizationArgs[i].value) {
```

Comment:
I'm not quite sure why you're introducing this "if" block ... wouldn't it be better to instead set this.tmpCustomizationArgs[i].value in line 422-428 instead of doing so for customizationArgsDics[caName] (similar to what you do for the video component) so that you can keep this part general.

Also reminder to please write some frontend tests for this, as requested previously, to ensure there aren't bugs in the logic.

Reply:
Done!